modules/packages/Sort.chpl

```diff
@@ -1906,8 +1908,10 @@ module TwoArrayPartitioning {
     var globalCounts:[0..#countsSize] int;
     var globalEnds:[0..#countsSize] int;
 
-    var distTasks:[0..-1] TwoArraySortTask;
-    var localTasks:[0..#numLocales] TasksForLocale;
+    // was [0..-1]
+    var distTasks: list(TwoArraySortTask);
+    // was [0..#numLocales]
+    var localTasks: list(TasksForLocale);
```

Comment:
I think this one should remain an array. I suspect you changed both fields named localTasks but this one is never appended to.